src/edu/csus/ecs/pc2/core/execute/SandboxResults.java

```diff
@@ -33,8 +39,14 @@ public SandboxResults(File propertiesFilename) throws IOException {
         resultsProperties.load(fileInputStream);
     }
 
-    public String getProperty(String s) {
```

Comment:
For some reason GitHub will not let me comment on certain lines in this file (probably because they have not been changed IN THIS VERSION).  This comment is about the `public SandboxResults()` constructor, above.
The existence of this no-arg constructor allows a developer to construct a SandboxResults object that has no results in it -- which is an object that exists in what is called an Invalid State.  This is in general, bad design; it should not ever be possible to have an object which is in an Invalid State.  (Yes, I know it's possible all over the existing code; that doesn't mean we should allow propagation of bad design).  This constructor should be eliminated; or, if it is required for some reason (like an IDE complaining if there's no no-arg constructor), it should be made private.

Reply:
Fixed.  Please state an action to take, like "Remove public SandboxResults() or make it private" then
go into an explanation. thanks.